integration-tests/binary/jls_15_13_1_2.out.exitcode

```diff
@@ -0,0 +1 @@
+terminated by signal (or crashed)
```

Comment:
doesn't seem to me like our test framework can recognize this (?)

Reply:
What do you mean?

Reply:
https://doc.rust-lang.org/std/process/struct.ExitStatus.html#method.code

Reply:
I thought only integers were allowed... ?

Reply:
So, is this about the test case being wrong? or about the integration test framework having a bug.

Reply:
The exitcode file is compared by string. So you can actually write to it what you want. In case the program terminates abnormally (and therefore does not return an exit code), the string shown above is printed by the test framework.